\documentclass[12pt]{article}
\usepackage[round]{natbib}
%\usepackage[T1]{fontenc}% change to nicer font
%\usepackage[bitstream-charter]{mathdesign}% change to nicer font
\usepackage[onehalfspacing]{setspace} % set spacing to one and a half
\usepackage{graphicx}%for graphics
%to add in test referencing of figs
\PassOptionsToPackage{hyphens,spaces,obeyspaces}{url}\usepackage{hyperref}%to avoid clashes between url and hyperref
\hypersetup{breaklinks=true,bookmarks=true,colorlinks=true,urlcolor=blue,linkcolor=blue,citecolor=blue,pdfauthor={Rupert A.\ Collins},pdftitle={Dowton et al. reply}}

% The following parameters seem to provide a reasonable page setup.
\topmargin 0.0cm
\oddsidemargin 0.2cm
\textwidth 16cm 
\textheight 21cm
\footskip 1.0cm

%adds new float type
\usepackage[labelsep=space,labelfont=bf,singlelinecheck=false]{caption}
\usepackage{float}
\DeclareCaptionType[fileext=flt]{Supplementary Data}


%The next command sets up an environment for the abstract to your paper.
\newenvironment{sciabstract}{%
\begin{quote} }
{\end{quote}}

% Include your paper's title here
\title{\bf Known knowns, known unknowns, unknown unknowns and unknown knowns in DNA barcoding: a reply to Dowton et al.} 


% Place the author information here.  Please hand-code the contact
\author
{Rupert A.\ Collins$^{1*}$\\
Robert H. Cruickshank$^{2}$\\
\\
\normalsize{$^{1}$Laborat\'orio de Evolu\c c\~ao e Gen\'etica Animal,}\\
\normalsize{Departamento de Biologia, Universidade Federal do Amazonas,}\\
\normalsize{Av.\ Rodrigo Ot\'avio, Manaus, Amazonas, Brasil}\\
\normalsize{$^{2}$Department of Ecology, Faculty of Agriculture and Life Sciences,}\\
\normalsize{Lincoln University,  Lincoln 7647,  Canterbury, New Zealand}\\
\\
\normalsize{$^\ast$To whom correspondence should be addressed; E-mail:  rupertcollins@gmail.com}
}

% Include the date command, but leave its argument blank.
\date{}

%%%%%%%%%%%%%%%%% END OF PREAMBLE %%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%% END OF PREAMBLE %%%%%%%%%%%%%%%%


\begin{document} 
% Make the title.
\maketitle 

% Place your abstract within the special {sciabstract} environment.
%\begin{sciabstract}
%\subsection*{Abstract}
%\end{sciabstract}

%\section*{Introduction}

%\section*{Materials and methods}
%
%
%\section*{Results}
%
%
%\section*{Discussion}
%
%\section*{Acknowledgements}

In a recent commentary, \citet{Dowton2014} propose a framework for “next-generation” DNA barcoding, whereby multi-locus datasets are coupled with coalescent-based species delimitation methods to make specimen identifications. They claim single-locus DNA barcoding is outdated, and a multilocus approach superior, with their assertions supported by an analysis of 33 species of \emph{Sarcophaga} flesh flies. Here, we reanalyse their data and show that a standard DNA barcode analysis is in fact capable of identifying 99.8\% (all but one) of their \emph{Sarcophaga} specimens, and that their conclusions misrepresent their data. We also discuss the benefits and drawbacks to their vision of ``next-gen'' barcoding.

DNA barcoding for identifying biological specimens relies on a reference library of sequences with \emph{a priori} identifications. Adult life stages can accurately identified using diagnostic morphological characters, and then the DNA barcodes from immature or incomplete specimens can be matched to these, a technique highly applicable to accurate forensic identification of Diptera larvae \citep{Meiklejohn2011}. However, the breadth and depth of the reference library has implications for the degree of identification success that can be obtained from it \citep{Bergsten2012,Virgilio2012,Zhang2012b}. Once the DNA barcode library has been assembled, the sequences contained and not contained in the library can be formulated as follows: ``known knowns'' (described and identified species with DNA barcodes in the library); ``known unknowns'' (described species without DNA barcodes in the library); ``unknown knowns'' (divergent lineages among the described species present in the library = possibly cryptic or overlooked species); and ``unknown unknowns'' (undescribed or cryptic species without DNA barcodes in the library). This formulation helps us characterise an incompletely sampled reference library.

Sequence data generated from query specimens can then identified using the library by sequence similarity matching \citep{Meier2006}, usually in a simulated scenario (database members acting as both reference library and query sequences). When there is 100\% genetic similarity between the query and a matching, unambiguous sequence(s), the identification is trivial. However, in the absence of complete haplotype sampling, non-identical sequences pose problems in differentiating intra- from inter-specific variation \citep{Virgilio2012}, and obtaining a correct identification or correct non-identification (if no match present for singletons) relies upon heuristic solutions. Early studies on DNA barcoding \citep[e.g.][]{Hebert2004} proposed a ``ten times'' rule to determine a threshold value of genetic distance---interspecific variation being ten times or more the intraspecific variation---but this was later discredited \citep{Hickerson2006}. More often, thresholds are generated by taking a database of specimens with a known taxonomy, and calculating the threshold which minimises the cumulative identification failure incorporating false positive error (no matches within the threshold value but conspecific species available) and false negative error (more than one species recorded within threshold) \citep{Meyer2005}. Based on this approach, the Barcode of Life Data System Barcode Index Number (BOLD BIN) system \citep{Ratnasingham2013} uses 2.2\% \emph{p}-distance threshold (with subsequent refinement using Markov clustering). However, in many studies, unjustified and arbitrary threshold values are frequently copied from the literature without the consideration of alternatives, a practice that was criticised by \citet{Collins2013}, despite the availability of easy-to-use tools to optimise thresholds empirically \citep{Brown2012,Puillandre2012,Virgilio2012,Sonet2013}.

\citeauthor{Dowton2014}\ report that single locus COI barcodes are able to identify just 27 of 31 \emph{Sarcophaga} species (87\%). When compared to the 100\% correctly identified using the multi-species coalescent (from herein ``MSC'') software BPP \citep{Yang2010}, this appears to validate their claim that multilocus datasets and highly parameterised analyses are indeed required. However, for their standard DNA barcode analysis they used a K2P divergence threshold of 4\%, but no references were provided to support the application of this value to their taxa. We reanalysed the data of \citeauthor{Dowton2014} (\url{http://dx.doi.org/10.5061/dryad.st467}) to establish if 4\% was an accurate threshold for distinguishing inter- and intra-specific variation. First, we used the taxonomic-name-based threshold optimisation technique \textsl{threshOpt} in the \textsc{Spider} package for R \citep{Brown2012}, followed by a name-free estimation of discontinuities in amalgamated genetic distance using the \textsl{localMinima} function also in \textsc{Spider} (full details to repeat our analysis are presented an R scripts in Supplementary Material). The two methods agreed on an optimum threshold of 2.13\% (\emph{p}-distance), and therefore lower than the 4\% used by \citeauthor{Dowton2014}. We then applied this optimised threshold to simulate the identification of the 405 specimens---from the 33 \emph{Sarcophaga} species used in their ``testing'' dataset---with the ``best close match'' (BCM) technique of \citep{Meier2006}.

We report that with the exception of four individuals, all 405 specimens were either correctly identified to species (if conspecifics were present in the library), or correctly reported as having no conspecific in the library (i.e.\ singletons).  Of the four misidentified specimens, one (KM592), considered as \emph{S}.\ \emph{bancroftorum} `clade 1' according to taxon labelling and Table 1.\ of \citeauthor{Dowton2014},  was identifed as \emph{S}.\ \emph{taenionota}. However, this appears to be simply a mislabelled sequence, as the same individual is labelled as  \emph{S}.\ \emph{taenionota} in Fig.\ 2.\ of \citet{Meiklejohn2013}. Three further specimens could not be identified despite having conspecifics in the library; it was proposed by \citet{Meiklejohn2012} that two of these DNA barcodes (\emph{S}.\ \emph{omikron} KM311; \emph{S}.\ \emph{spinigera} KM260) could represent paralogous nuclear sequences, which might explain the large divergence from their conspecifics. However, upon examining the alignment and the trace files on BOLD, a more simple explanation is apparent, whereby sequencing errors due to ``dye blobs'' have introduced an additional nucleotide, thus frame-shifting part of the alignment and causing the large divergences. When this is corrected, the sequences are identified correctly (Supplementary Material).  The remaining individual (\emph{S}.\ \emph{australis} KM673) appears to be the only genuine case of an unidentifiable individual, being 3.18\% divergent from its closest conspecific. Thus, excluding the probable errors, this represents a 99.8\% success rate, and calls into question the 87\% success rate reported by \citeauthor{Dowton2014}\ as justification for the superiority of the MSC. Importantly, the single individual that could not be placed (\emph{S}.\ \emph{australis} KM673) was not included in the BPP analysis of \citeauthor{Dowton2014}, so it cannot currently be established if this specimen would have been included or not as the correct species by the MSC approach. 

This discrepancy between success rates reported here and in \citeauthor{Dowton2014}\ is due to two factors: (1) the identification method was applied to species rather than individuals, a general problem in much of the DNA barcoding literature whereby species discovery is conflated with specimen identification \citep{Collins2013}; and (2) the use of an arbitrary 4\% threshold which compromised identification ability due to unnecessarily high error: 6.78\% cumulative rate for the 4\% threshold in contrast to 0.23\% for the 2.3\% threshold. 

%To provide a wider context we provide an additional analysis of 27,297 COI sequences from BOLD. Birds were chosen as their taxonomy is arguably the most mature of all animals and therefore are likely to show the strongest correspondence between nomenclature and genetic structure \citep{Hebert2004}. After filtering we were left with 20,011 barcodes from 3,665 species and 1,320 genera. These were subjected to the threshold optimisation as described previously. We report that the optimum threshold was much lower than 4\% at 0.6\%, and in agreement with analyses of \citep{Ratnasingham2013} for North American birds (0.7\%). When applied to these data, a 4\% threshold yields a false-negative rate of 42\% (compared to 12\% for the optimum threshold), and an ID success rate of 88.2\% under the BCM. Thus, a considerable number of closely related bird species have a divergence of less than 4\%, and applying such a threshold would likely result in unacceptably high rates of false negative error for query sequences not represented in the library.

 The aim here is to illustrate that there is no ``one-size-fits-all'' threshold for all taxa, and optimising thresholds based on the library is a straightforward way to dramatically increase the ability of DNA barcodes to identify specimens. This was reiterated by \citep{Ratnasingham2013}, who state: ``performance, as measured by the number of correctly recognized species, dropped steeply when the threshold deviated on either side of optimality''.

Regardless, an optimised threshold while superior to an arbitrary one such as 4\%, remains nevertheless a heuristic approach that averages---albeit a more accurate average---over all species in that lineage, and will depend heavily on the depth and breadth of sampling and the maturity of the taxonomy upon which the thresholds are derived. Where there are many species with multiple divergent lineages within them---due to either incomplete taxonomy, cryptic speciation, or geographically isolated populations---intraspecific divergences may be overestimated by an approach to threshold optimisation that relies on taxonomic names. In this regard we see the potential that MSC methods can offer if applied \emph{a priori} to the dataset to derive posterior probabilities of speciation events among a putatively single species (i.e. turning ``unknown knowns'' into ``known knowns''). The advantage of an MSC approach is that it helps provide a strong biological basis for recognising this branch length variation as potential speciation events rather than an artifacts such as geographic sampling bias \citep{Bergsten2012}. However, it is questionable whether such statistics would be reliable where they would be most useful---i.e.\ for singletons---due to the sampling problems with rarity in species delimitation \citep{Lim2012}.

A MSC approach does have the potential to improve on current DNA barcoding practices in the case of haplotype mixtures due to interspecific hybridisation and incomplete lineage sorting. However, before investing resources, it is worth estimating how prevalent these processes/patterns really are. A recent study \citep{Ross2014} re-assessed the findings of an influential paper \citep{Funk2003} stating that species-level mitochondrial paraphyly occurs in roughly 23\% of animal species. \citeauthor{Ross2014} reported lower, but similar levels (19\%) based on a larger dataset. As to the causes of this non-monophyly, in a review of bird data \citep{McKay2010} reported 14.3\% of bird species as non-monophyletic, with 6\% non-monophyletic due to either introgression or incomplete lineage sorting, and 8\% due to uncertain taxonomy. Misidentified specimens have also been cited as a potential problem particularly in DNA barcode data, and especially for taxa where multiple workers are submitting data for the same taxa independently \citep{Collins2013}. 

So, in which situations would the MSC be superior? In the absence of integrative character data, it is unlikely that MSC methods will be successful in resolving the cases of uncertain taxonomy because of the unavailability of node-based names derived from these kinds of analyses \citep{Bauer2010}. Likewise, misidentified voucher specimens are in most cases just as likely to be detected with single locus markers as with multiple \citep{Becker2011,Ko2013}. On the other hand, in the cases of introgression and incomplete lineage sorting, an MSC approach would in theory be superior to single locus barcoding. However, the question is whether the problem has been quantified sufficiently to demand a new paradigm of specimen identification, and whether current methods/data could be improved or adapted without recourse to ``overkill''. As an example, there is still a widespread assumption that reciprocal monophyly is required for DNA barcode identifications \citep{Goldstein2011}, but this is not the case \citep{Meier2008}, and correct identifications are possible as long as haplotypes are not shared \citep{Meier2006}. There are also several identification methods now available for single-loci that are substantially more sophisticated than sequence matching or tree building, and these may also offer improvements: \citep{Nielsen2006} and \citep{Abdo2007} developed coalescent theory to obtain statistical evidence for species membership; \citep{Zhang2008} and \citep{Zhang2012a} used machine learning and neural network methods; \citep{Zhang2012b} used fuzzy-set theory; and \citep{Weitschek2013} use a character-based logic method. All of these techniques make different assumptions resulting in specific advantages and disadvantages, but they have unfortunately been largely ignored in favour of the simpler methods.

In a practical sense, MSC methods are incredibly computationally intensive with genomic scale data. Even with the minimum of two genes and a very modest dataset of just 136 individuals, \citeauthor{Dowton2014}\ reported that the analysis remained ``prohibitively slow''. Due to the lengthy exploratory analyses and inevitable model and prior selection issues when running highly parameterised species-tree analyses with hundreds of loci, we are highly doubtful that this vision of DNA barcoding can being used in the type of routine identifications where speed is important \citep{Armstrong2005}. Rather than concentrating on analytical methods, perhaps it is better to recognise that comprehensive sampling and complete reference libraries \citep{Virgilio2012,Boykin2012}, bring arguably the single biggest improvement to DNA barcode identification success \citep{Ekrem2007}.

Ultimately, researchers will develop the best methods for their questions and study system, so if MSC methods are required for some difficult taxa then they should be used \citep{Dupuis2012}. For forensic and legal applications in particular, the application of posterior probabilities from multilocus data is a more robust, and certainly more attractive approach. However, it is important to note that generating extra data and performing additional analyses on a case-by-case basis is not DNA barcoding. The power of DNA barcoding relies in its standardisation and the subsequent scalability associated with that. To achieve standardisation for genomic data, there are considerable costs associated with re-sequencing the hundreds of thousands of specimens already barcoded, as well as in processing/hosting this data. \citep{Taylor2012} also suggested that post-Sanger genomic-sequencing technologies have rendered DNA barcoding obsolete. Post-Sanger sequencing has certainly increased the cost efficiency of generating nucleotide data---and it can readily be used to generate barcode libraries \citep{Shokralla2014}---but in our opinion the more significant costs of DNA barcoding come with collection, pre-laboratory processing, identification, databasing, and curation of the vouchers in a collection \citep{Borisenko2009,Gregory2005,Puillandre2012}. 

In conclusion, our opinion is that DNA barcoding need not assume ``gene trees and species trees are synonymous'', but rather that DNA barcodes are an effective proxy for species that can achieve high rates of identification success when appropriate methods are used to analyse them. So, while it is important to know where the limitations of the method lies, it is also important to realise its strengths. There should be a good justification for abandoning a proven technique, and the case study presented by \citeauthor{Dowton2014}\ does not sufficiently demonstrate the failure of standard DNA barcoding due to the choice of a poorly performing identification technique.  We reiterate the prevailing orthodoxy as questioned by \citep{Karl2012}, namely that (1) ``more data are always better'', and that (2) ``one needs to do a Bayesian analysis''. Therefore, we agree that researchers should be looking into ``smarter'' methods for taxon identification, but only in cases where the ``dumb'' methods have first been comprehensively shown not to work.

\bibliography{dowton_reply}%need to manually add doi: downton, shokralla, bauer

\bibliographystyle{apa-good_sysbio}

%%%% start the figs and supplementary data
\clearpage
\newpage

%suppl 1
\begin{Supplementary Data}
\caption{Bash shell script. }
\label{suppl:bash}
\end{Supplementary Data}

%suppl 2
\begin{Supplementary Data}
\caption{R script. }
\label{suppl:R}
\end{Supplementary Data}


\end{document}