\documentclass[a4paper,12pt]{article}
\usepackage[round]{natbib}%ref formats
\usepackage[onehalfspacing]{setspace} % set spacing to one and a half
\usepackage{lineno}%line numbers
%to add in test referencing of figs
\PassOptionsToPackage{hyphens,spaces,obeyspaces}{url}\usepackage{hyperref}%to avoid clashes between url and hyperref
\hypersetup{breaklinks=true,bookmarks=true,colorlinks=true,urlcolor=blue,linkcolor=blue,citecolor=blue,
pdfauthor={Rupert A.\ Collins, Robert H.\ Cruickshank},
pdftitle={Known knowns, known unknowns, unknown unknowns and unknown knowns in DNA barcoding: a comment on Dowton et al.},
pdfkeywords={DNA barcoding, Multi-species coalescent, Identification success, Threshold optimisation},
pdfsubject={Systematic Biology 10.1093/sysbio/syu060}}

% The following parameters seem to provide a reasonable page setup.
\topmargin 0.0cm
\oddsidemargin 0.2cm
\textwidth 16cm 
\textheight 21cm
\footskip 1.0cm

%reformat the section headings
\usepackage{titlesec}
\titleformat{\section}{\scshape\normalsize\centering}{\thesection}{1em}{}

%The next command sets up an environment for the abstract to your paper.
\newenvironment{sciabstract}{%
\begin{quote} }
{\end{quote}}

% Include your paper's title here
\title{\bf Known knowns, known unknowns, unknown unknowns and unknown knowns in DNA barcoding: a comment on Dowton et al.} 


% Place the author information here.  Please hand-code the contact
\author
{Rupert A.\ Collins$^{1*}$\\
Robert H.\ Cruickshank$^{2}$\\
\\
\normalsize{$^{1}$Laborat\'orio de Evolu\c c\~ao e Gen\'etica Animal,}\\
\normalsize{Departamento de Biologia, Universidade Federal do Amazonas,}\\
\normalsize{Av.\ Rodrigo Ot\'avio, Manaus, Amazonas, Brasil}\\
\normalsize{$^{2}$Department of Ecology, Faculty of Agriculture and Life Sciences,}\\
\normalsize{Lincoln University,  Lincoln 7647,  Canterbury, New Zealand}\\
\\
\normalsize{$^\ast$To whom correspondence should be addressed; E-mail:  rupertcollins@gmail.com}
}

% Include the date command, but leave its argument blank.
\date{}

%%%%%%%%%%%%%%%%% END OF PREAMBLE %%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%% END OF PREAMBLE %%%%%%%%%%%%%%%%


\begin{document} 
% Make the title.
\maketitle 
% Add line numbering
%\linenumbers

In a recent commentary, \citet{Dowton2014} propose a framework for ``next-generation'' DNA barcoding, whereby multi-locus datasets are coupled with coalescent-based species delimitation methods to make specimen identifications. They claim single-locus DNA barcoding is outdated, and a multilocus approach superior, with their assertions supported by an analysis of 33 species of \emph{Sarcophaga} flesh flies. Here, we reanalyse their data and show that a standard DNA barcode analysis is in fact capable of identifying 99.8\% (all but one) of their \emph{Sarcophaga} specimens, and that their conclusions misrepresent their data. We also discuss the benefits and drawbacks to their vision of ``next-gen'' barcoding.


\section*{Identifying Biological Specimens using DNA Barcodes}

DNA barcoding for identifying biological specimens relies on a reference library of sequences with \emph{a priori} identifications. Adult life stages are identified using diagnostic morphological characters, and then the DNA barcodes from immature or incomplete specimens can be matched to these, a technique highly applicable to accurate forensic identification of Diptera larvae \citep{Meiklejohn2011}. However, the breadth and depth of the reference library has implications for the degree of identification success that can be obtained from it \citep{Bergsten2012,Virgilio2012,Zhang2012b}. Once the DNA barcode library has been assembled, the sequences contained and not contained in the library can be formulated as follows: ``known knowns'' (described and identified species with DNA barcodes in the library); ``known unknowns'' (described species without DNA barcodes in the library); ``unknown knowns'' (divergent lineages among the described species present in the library = possibly cryptic or overlooked species); and ``unknown unknowns'' (undescribed or cryptic species without DNA barcodes in the library). This formulation helps us characterise an incompletely sampled reference library.

Sequence data generated from query specimens can then be identified using the library by sequence similarity matching \citep{Meier2006}, usually in a simulated scenario (database members acting as both reference library and query sequences). When there is 100\% genetic similarity between the query and a matching, unambiguous sequence(s), the identification is trivial. However, in the absence of complete haplotype sampling, non-identical sequences pose problems in differentiating intra- from inter-specific variation \citep{Virgilio2012}, and obtaining a correct identification or correct non-identification---if no match is present for singletons---relies upon heuristic solutions. Early studies on DNA barcoding \citep[e.g.][]{Hebert2004} proposed a ``ten times'' rule to determine a threshold value of genetic distance---interspecific variation being ten times or more the intraspecific variation---but this was later discredited \citep{Hickerson2006}. More often, thresholds are generated by taking a database of specimens with a known taxonomy, and calculating the threshold that minimises the cumulative identification failure incorporating false positive error (no matches within the threshold value but conspecific samples available) and false negative error (more than one species recorded within threshold) \citep{Meyer2005}. Based on this approach, the Barcode of Life Data System (BOLD) Barcode Index Number (BIN) algorithm \citep{Ratnasingham2013} uses 2.2\% \emph{p}-distance threshold (with subsequent refinement using Markov clustering). However, in many studies, unjustified and arbitrary threshold values are frequently copied from the literature without the consideration of alternatives---a practice that was criticised by \citet{Collins2013}---despite the availability of easy-to-use tools to optimise thresholds empirically \citep{Brown2012,Puillandre2012b,Virgilio2012,Sonet2013}.


\section*{A Reanalysis of Dowton et al.}

\citet{Dowton2014} (from herein \citeauthor{Dowton2014}) reported that single locus COI barcodes were able to identify just 27 of 31 \emph{Sarcophaga} species (87\%). When compared to the 100\% correctly identified using the multi-species coalescent (from herein ``MSC'') software BPP \citep{Yang2010}, this appears to validate their claim that multilocus datasets and highly parameterised analyses are indeed required. However, for their standard DNA barcode analysis they used a K2P divergence threshold of 4\%, but no references were provided to support the application of this value to their taxa. We reanalysed the data of \citeauthor{Dowton2014}\ (\url{http://dx.doi.org/10.5061/dryad.st467}) to establish if 4\% was an accurate threshold for distinguishing inter- and intra-specific variation. First, we used the taxonomic-name-based threshold optimisation technique \textsl{threshOpt} in the \textsc{Spider} package for R \citep{Brown2012}, followed by a name-free estimation of discontinuities in amalgamated genetic distances using the \textsl{localMinima} function, also in \textsc{Spider} (full details to repeat our analysis is presented as an R script in Supplementary Material). The two methods agreed on an optimum threshold of 2.13\% (\emph{p}-distance), and therefore lower than the 4\% used by \citeauthor{Dowton2014} We then applied this optimised threshold to simulate the identification of the 405 specimens---from the 33 \emph{Sarcophaga} species used in their ``testing'' dataset---with the ``best close match'' (BCM) technique of \citet{Meier2006}.

We report that 401 of 405 specimens were either correctly identified to species (if conspecifics were present in the library), or correctly reported as having no conspecific in the library (i.e.\ singletons).  Of the four misidentified specimens, one (KM592)---considered as \emph{S}.\ \emph{bancroftorum} `clade 1' according to the taxon labelling in the dataset provided and Table 1.\ of \citeauthor{Dowton2014}---was identified here as \emph{S}.\ \emph{taenionota}. However, this appears to be simply a mislabelled sequence, as the same individual is labelled as  \emph{S}.\ \emph{taenionota} in Fig.\ 2.\ of \citet{Meiklejohn2013}. Three further specimens could not be identified despite having conspecifics in the library; it was proposed by \citet{Meiklejohn2012} that sequences from two of these specimens (\emph{S}.\ \emph{omikron} KM311 and  \emph{S}.\ \emph{spinigera} KM260) could represent paralogous nuclear copies, which might explain the large divergences from their closest conspecifics (5.73\% and 2.51\% respectively). However, upon examining the alignment and the trace files on BOLD, a more simple explanation is apparent, whereby sequencing errors due to ``dye blobs'' have introduced an additional nucleotide, thus frame-shifting part of the alignment and causing the divergences. When these errors are corrected, the sequences are identified correctly (Supplementary Material).  The remaining individual (\emph{S}.\ \emph{australis} KM673) appears to be the only genuine case of an unidentifiable individual, being 3.35\% divergent from its closest putative conspecific. Thus, excluding the probable errors, this represents a 99.8\% success rate, and calls into question the 87\% success rate reported by \citeauthor{Dowton2014}\ as justification for the superiority of the MSC. Importantly, the single individual that could not be placed (\emph{S}.\ \emph{australis} KM673) was not included in the BPP analysis of \citeauthor{Dowton2014}, so it cannot currently be established if this specimen would have been included or not as the correct species by the MSC approach. 

This discrepancy between success rates reported here and in \citeauthor{Dowton2014}\ is due to two factors: (1) the identification method was applied to species rather than individuals, a general problem in much of the DNA barcoding literature whereby species discovery is conflated with specimen identification \citep{Collins2013}; and (2) the use of an arbitrary 4\% threshold which compromised identification ability due to higher potential error: 16.09\% cumulative rate for the 4\% threshold in contrast to 10.57\% for the 2.13\% threshold (error rates for full dataset, not just the 33 query species). Therefore, the aim here is to illustrate that there is no ``one-size-fits-all'' threshold for all taxa, and optimising thresholds based on the library is a straightforward way to dramatically increase the ability of DNA barcodes to identify specimens. This was reiterated by \citet{Ratnasingham2013}, who stated: ``performance, as measured by the number of correctly recognized species, dropped steeply when the threshold deviated on either side of optimality''.


\section*{Pros and Cons of Multilocus Methods}

Regardless, an optimised threshold while superior to an arbitrary one such as 4\%, remains nevertheless a heuristic approach that averages---albeit a more accurate average---over all species in that dataset, and will depend upon substitution-rate variation among lineages, the depth and breadth of taxon sampling, and the maturity of the taxonomy upon which the thresholds are derived. Where there are many species with multiple divergent lineages within them---due to either incomplete taxonomy, cryptic speciation, or geographically isolated populations---intraspecific divergences may be overestimated by an approach to threshold optimisation that relies on taxonomic names. In this regard we see the potential that MSC methods can offer if applied \emph{a priori} to the dataset to derive posterior probabilities of speciation events within a putatively single species (i.e.\ turning ``unknown knowns'' into ``known knowns''). The advantage of an MSC approach is that it helps provide a stronger biological basis for recognising this branch length variation as a potential speciation event rather than an artefact due to geographic sampling bias \citep{Bergsten2012}. However, it is questionable whether such statistics would be reliable where they would be most useful---i.e.\ for singletons such as \emph{S}.\ \emph{australis} KM673---due to the sampling and parameter estimation problems associated with taxon rarity in species delimitation  methods \citep{Lim2012}.

An MSC approach does have the potential to improve on current DNA barcoding practices in the case of mixed clusters of haplotypes due to non-monophyly. However, before investing resources, it is worth estimating how prevalent these processes/patterns really are. A recent study \citep{Ross2014} re-assessed the findings of an influential paper \citep{Funk2003} stating that species-level mitochondrial paraphyly occurs in roughly 23\% of animal species; he reported lower, but similar levels (19\%) based on a larger dataset. As to the causes of this non-monophyly, a review of bird data \citep{McKay2010} reported 14.3\% of bird species as non-monophyletic, with 6\% non-monophyletic due to either introgression or incomplete lineage sorting, and 8\% due to uncertain taxonomy. Misidentified specimens have also been cited as a potential problem particularly in DNA barcode data, and especially for taxa where multiple workers are submitting data for the same taxa independently \citep{Collins2013}. 

So, in which of these situations would the MSC be superior? As stated previously, the MSC can be applied to robustly delimit species boundaries, but in the absence of integrative character data it is unlikely to be successful in completely resolving cases of uncertain taxonomy, due to the unavailability of node-based names derived from these kinds of analyses \citep{Bauer2010}. Likewise, misidentified voucher specimens are in most cases just as likely to be detected with single locus markers as with multiple \citep{Becker2011,Ko2013}. On the other hand, in the cases of introgression and incomplete lineage sorting, an MSC approach would in theory be superior to single locus barcoding. However, including a single nuclear gene as carried out by \citeauthor{Dowton2014}, is unlikely to provide much additional information as the power of nuclear loci generally lies in their multiplicity \citep{Edwards2009}. This is confirmed by our analysis of \citeauthor{Dowton2014}'s CAD data (Supplementary Material); only 67\% of the total species were monophyletic, and the BCM identification success of their 33 test species was just 51\% for this gene. 


\section*{Practical Issues}

In a practical sense, MSC methods are incredibly computationally intensive with genomic scale data. Even with the minimum of two genes and a very modest subset of just 136 individuals, \citeauthor{Dowton2014}\ reported that the analysis remained ``prohibitively slow'', and they were unable to analyse all their individuals as queries, only testing a random single individual. Due to the lengthy exploratory analyses and inevitable model and prior selection issues when running highly parameterised species-tree analyses with hundreds of loci, we are highly doubtful that this vision of DNA barcoding can be used for the types of routine identification where speed is important \citep{Armstrong2005}. Of course, we recognise that the field should not disregard philosophical and theoretic advances due to purely practical issues such as computational burden, but it seems that despite the increased availability of cloud computing facilities, there appears to be an even greater increase in the volumes of data now generated by genomic sequencing. Clearly, until considerably faster species delimitation methods can be developed, a pragmatic attitude cannot be entirely avoided.  Therefore, in order to benchmark the efficiency and accuracy of species delimitation methodologies, it should now be a priority to highlight exemplar datasets---empirical and/or simulated---for which MSC methods clearly outperform simpler mtDNA analyses. Although this was one of the intentions of \citeauthor{Dowton2014}, our results show that the dataset they used does not represent such a demonstration. 


\section*{Improvements on Current Methods}

Despite some clear theoretical advantages of MSC techniques in certain situations, the question remains as to whether the problems with mtDNA data have been quantified sufficiently to demand a new paradigm of specimen identification, and whether current methods/data could be improved or adapted without recourse to ``overkill''. As an example, there is still a widespread assumption that reciprocal monophyly is required for DNA barcode identifications \citep{Goldstein2011}, but this is not the case \citep{Meier2008}, and correct identifications are possible as long as haplotypes are not shared \citep{Meier2006}. There are also several identification methods now available for single loci that are substantially more sophisticated than sequence matching or tree building, and these may also offer improvements: \citet{Nielsen2006} and \citet{Abdo2007} developed coalescent theory to obtain statistical evidence for species membership; \citet{Zhang2008} and \citet{Zhang2012a} used machine learning and neural network methods; \citet{Zhang2012b} used fuzzy-set theory; and \citet{Weitschek2013} employed a character-based logic method. All of these techniques make different assumptions, resulting in specific advantages and disadvantages, but they have unfortunately been largely ignored in favour of the simpler tree-building methods. However, rather than concentrating on analytical methods, perhaps it is better to recognise that comprehensive sampling and complete reference libraries \citep{Boykin2012,Virgilio2012}, bring arguably the single biggest improvement to DNA barcode identification success \citep{Ekrem2007}. It is also important not to forget that the  voucher specimens can also be examined for morphological characters in cases of ambiguity (although this may not be appropriate for incomplete or juvenile specimens).


\section*{Conclusions and Future Directions}

Ultimately, researchers will develop the best methods for their questions and study system, so if MSC methods are required for some difficult taxa then they should be used \citep{Dupuis2012}. For forensic and legal applications in particular, posterior probabilities from multilocus data offer a more robust and certainly more attractive approach. However, it is important to note that generating extra data and performing additional analyses on a case-by-case basis is not DNA barcoding. The power of DNA barcoding relies in its standardisation and the subsequent scalability associated with that. To achieve standardisation for genomic data, there are considerable costs associated with re-sequencing the millions of specimens already barcoded, as well as in curating this data. \citet{Taylor2012} also suggested that post-Sanger genomic-sequencing technologies have rendered DNA barcoding obsolete. Parallel sequencing has certainly increased the cost efficiency of generating nucleotide data---and it can readily be used to generate barcode libraries \citep{Shokralla2014}---but in our opinion the more significant costs of DNA barcoding come with collection, pre-laboratory processing, identification, databasing, and curation of the vouchers in a collection \citep{Gregory2005,Borisenko2009,Puillandre2012a}. Perhaps a more tractable option to better capitalise on new sequencing technologies is through the generation of whole mitochondrial genomes \citep[see][]{Gillett2014}. The additional gene sampling may reduce stochastic error and improve resolution of some closely related taxa, but remain computationally efficient due to mtDNA being effectively a single locus. Furthermore, these data would remain compatible with the existing DNA barcode system, and can also be readily reused for phylogenetic purposes \citep{Gillett2014}.

In conclusion, our opinion is that DNA barcoding need not assume ``gene trees and species trees are synonymous'', but rather that DNA barcodes are an effective proxy for species that can achieve high rates of identification success when appropriate methods are used to analyse them. So, while it is important to know where the limitations of the method lies, it is also important to realise its strengths. There should be a good justification for abandoning a proven technique, and the case study presented by \citeauthor{Dowton2014}\ does not sufficiently demonstrate the failure of standard DNA barcoding due to the choice of a poorly performing identification technique.  We reiterate \citet{Karl2012} in questioning the prevailing orthodoxy, namely that ``more data are always better'', and ``one needs to do a Bayesian analysis''. Therefore, we agree that researchers should be looking into ``smarter'' methods for taxon identification, but only in cases where the ``dumb'' methods have first been comprehensively shown not to work.


\section*{Supplementary Material}

Data available from the Dryad Digital Repository: \url{http://dx.doi.org/10.5061/dryad.h3g63}


\section*{Funding}

This work was supported by a CNPq Ci\^encia sem Fronteiras fellowship (400813/2012-2) for Rupert Collins.

\bibliography{dowton_reply}
\bibliographystyle{apa-good_sysbio}

\end{document}